### ISSUE TITLE
Transition doc over to the new `dart` tool

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/site-www#2617]
Although the existing Dart CLI tools will be around for quite a while, we need to transition over to the [new `dart` tool](https://dart.dev/tools/dart-tool):

1. Find pages that use the command line, and substitute the new `dart` tool for existing tools.
2. Convert the build and CI to use the new `dart` tool. (See #2623, #2624, and [this comment](https://github.com/dart-lang/site-www/issues/2617#issuecomment-807693733))

Separately, we'll need to transition some of the non-pub command reference pages over to new URLs (#2719). But for now, **keep using the old URLs**, and avoid editing the pages to be redirected.

Here's a list (a reordered version of the output of ``fgrep -Rl "\`\`\`terminal" src``) of the 38 pages we should look at and eventually update (there might be more that don't show up due to improper formatting): 

***Update: All known occurrences in pages are fixed! We should open new issues for anything else that turns up.***

- [x]  src/tools/pub/cmd/pub-deps.md (fixed by #3116)
- [x] src/tools/experiment-flags.md (fixed by #3118)
- ~~src/tools/pub/cmd/pub-run.md~~  (covered by #2719) 
- [x] ~~_Assigned to @KevinRohn:_ src/tools/pub/cmd/pub-outdated.md~~ (fixed by #2789)
- [x] ~~_Assigned to @KevinRohn:_ src/tools/pub/cmd/pub-upgrade.md~~ (fixed by #2825)
- [x] ~~_Assigned to @chethanprabhakar:_ src/tools/pub/cmd/pub-downgrade.md~~ (fixed by #2847)
- [x] ~~_Assigned to @Abhishek01039:_ src/tools/pub/cmd/pub-global.md~~ (fixed by #2843)
- [x] ~~src/tools/pub/cmd/pub-lish.md~~ (fixed by #2816)
- [x] ~~src/tools/pub/cmd/pub-uploader.md~~ (fixed by #2883)
- [x] ~~_Assigned to @KevinRohn:_ src/tools/pub/cmd/pub-cache.md~~ (fixed by #2825)
- [x] ~~_Assigned to @KevinRohn:_ src/tools/pub/cmd/pub-get.md~~ (fixed by #2826)
- [x] ~~src/tools/pub/troubleshoot.md~~ (fixed by #2854)
- [x] ~~src/tools/pub/publishing.md~~ (fixed by #2816)
- ~~src/tools/dartdoc.md~~ (no change needed... yet)
- ~~src/tools/dart2native.md~~ (covered by #2719)
- ~~src/_articles/libraries/dart-io.md~~ (we aren't updating articles)
- [x] ~~_Assigned to @shubhanshu02:_ src/_tutorials/web/get-started.md~~ (fixed by #2795)
- [x] ~~_Assigned to @shubhanshu02:_ src/_tutorials/libraries/shared-pkgs.md~~ (fixed by #2839)
- [x] ~~_Assigned to @shubhanshu02:_ src/_tutorials/server/get-started.md~~ (fixed by #2725)
- [x] ~~_Assigned to @shubhanshu02:_ src/_tutorials/server/httpserver.md~~ (fixed by #2848)
- [x] ~~_Assigned to @shubhanshu02:_ src/_tutorials/server/cmdline.md~~ (fixed by #2757)
- [x] ~~src/web/debugging.md~~ (fixed by #2850)
- ~~src/null-safety/index.md~~ (no terminal examples present as of 1/7)
- [x] ~~_Assigned to @shubhanshu02:_ src/_guides/libraries/c-interop.md~~ (fixed by #2924)
- ~~src/_guides/language/language-tour.md~~ (@kwalrath will handle)
- [x] _Assigned to @Sameerkash:_ src/_guides/language/formatting.md
- [x] _Assigned to @Sameerkash:_ src/_guides/packages.md
- [x] _Assigned to @Sameerkash:_ src/_includes/get-sdk.md
- [x] ~~_Assigned to @woinbo:_ src/tools/dart-devtools.md~~ (fixed by #2841)
- [x] ~~src/tools/build_runner.md~~ (fixed by #2877)
- ~~src/tools/dart-vm.md~~ (covered by #2719)
- ~~src/tools/dartanalyzer.md~~ (covered by #2719)
- ~~src/tools/dart-tool.md~~ (@kwalrath will handle)
- ~~src/tools/dartfmt.md~~ (covered by #2719)
- ~~src/tools/dartaotruntime.md~~ (@kwalrath will handle)
- ~~src/tools/dart2js.md~~ (@kwalrath will handle)
- [x] ~~src/tools/webdev.md~~ (fixed by #2870)
- ~~_Assigned to @shubhanshu02:_ src/tools/sdk/_linux.md~~ (no change needed)
- [x] ~~_Assigned to @woinbo:_ src/tools/sdk/_windows.md~~ (fixed by #2853)
- ~~_Assigned to @Sameerkash:_ src/tools/sdk/_mac.md~~ (no change needed)

/cc @johnpryan @mit-mit

**Comment by felpsisonfire** [comment]
Can I get the first 8? 

Also, I have a question: Do we need to substitute the entire Dart CLI tools with the new ones? Just to clarify
In the given example:
Actual:
```terminal
$ dartfmt -w bin lib test
```
New:
```terminal
$ dart format -o write bin lib test
``` 
The {% comment %} piece seems to have no effect in the page after the livereload

**Comment by felpsisonfire** [comment]
Just to get the things straight. We need to replace the old Dart CLI tools with the new dark tool, is that right? Using the given example:
Actual:
```terminal
$ dartfmt -w bin lib test
```
New:
```terminal
$ dart format -o write bin lib tes
```

**Comment by mit-mit** [comment]
A couple of PRs for the CI part here:

https://github.com/dart-lang/site-www/pull/2623
https://github.com/dart-lang/site-www/pull/2624

**Comment by kwalrath** [comment]
@felpsisonfire you've got it.

**Comment by felpsisonfire** [comment]
> @felpsisonfire you've got it.

Cool! Can I get the first 8?

**Comment by kwalrath** [comment]
> Cool! Can I get the first 8?

Absolutely. Thank you!

**Comment by sumandipanshu** [comment]
Can i work on this issue?
Also, what should i do for those `.md` files which are combined to form a single `.md` file.
For example: `dart2js` and `dart2native`, these two commands have been merged into a single command `dart compile`.

**Comment by sumandipanshu** [comment]
Also, I am confused about what actually we have to do here.
    1. Find pages that use the command line, and substitute the new dart tool for existing tools.
    2. Perhaps we could add a comment like the following to each occurrence of a command.

Which one we are supposed to do?

**Comment by kwalrath** [comment]
@sumandipanshu: Good questions. Now that the `dart` command is out of technical preview, testing usage isn't the focus any more. I suspect we'll want to start actually updating pages (and testing them and the commands along the way). But I'm not sure about the timing.

I'll ask tomorrow in our weekly meeting and update this bug to reflect what we decide.

**Comment by kwalrath** [comment]
We're going to take a slower approach than I expected to update these pages. I'll unblock this issue and #2678 once we're ready to go.

**Comment by kwalrath** [comment]
OK, this issue is unblocked! @sumandipanshu feel free to update (and test!) the commands. E.g., if you take `src/_guides/language/formatting.md`, then you'll do the following:

1. `dartfmt` -> `dart format` everywhere
2. Change the example command, if necessary, to one that works (`dart format -w .`?)

@felpsisonfire could you revisit #2629, and make sure all the new commands work? Thank you!

**Comment by shubhanshu02** [comment]
Hi @kwalrath! I'm a new contributor here. Can I work on `src/_tutorials/`?

**Comment by kwalrath** [comment]
Thanks for the offer, @shubhanshu02! I've assigned you https://github.com/dart-lang/site-www/blob/master/src/_tutorials/server/get-started.md (https://dart.dev/tutorials/server/get-started).

**Comment by shubhanshu02** [comment]
@kwalrath, I have two questions on the `src/_tutorials/server/get-started.md` file - 
1. Since `stagehand` is replaced by `dart create`, there is no need to install `stagehand`. So what should be done to the "Get more command-line developer tools" heading which has the `pub global activate stagehand`?
2. Also I noticed that `dart create` automatically runs `dart pub get` on completion. So should the "Get the app’s dependencies" heading be removed?

**Comment by kwalrath** [comment]
1. We won't need stagehand, so delete that heading and content. (You might have to adjust other text accordingly.)
2. Yes, I think you can remove that heading, but do update the description to say what `dart create` gets the packages you depend on. 

Thank you! It's good to see that you're testing this well!

**Comment by shubhanshu02** [comment]
> 1. We won't need stagehand, so delete that heading and content. (You might have to adjust other text accordingly.)
> 2. Yes, I think you can remove that heading, but do update the description to say that `dart create` gets the packages you depend on.
> 
> Thank you! It's good to see that you're testing this well!

Thank you @kwalrath for the help. I have created the PR, please review it.

**Comment by kwalrath** [comment]
@shubhanshu02 are you interested in adopting another page?

**Comment by shubhanshu02** [comment]
> @shubhanshu02 are you interested in adopting another page?

Yes, sure.

**Comment by kwalrath** [comment]
> > @shubhanshu02 are you interested in adopting another page?
> 
> Yes, sure.

How about https://dart.dev/tutorials/server/cmdline?

**Comment by KevinRohn** [comment]
Hi @kwalrath 

I would like to start contributing to the [pub tool](https://dart.dev/tools/pub/cmd) sites. I took a look at the website [pub outdated] (https://dart.dev/tools/pub/cmd/pub-outdated) and have some open questions.

1.) What should I do if a command output suggests using another command with the deprecated syntax?

For example:
```
$ dart pub upgrade 
Resolving dependencies... 
...
No dependencies changed.
1 package has newer versions incompatible with dependency constraints.
Try `pub outdated` for more information.
```
Should I open an issue to fix the static output information to change `pub outdated` to `dart pub outdated` ?

2.) Is the whole tool rename or should I just focus on the correct syntax? 
In regarding to this question, if the sentence describes the *pub tool*, should it be changed to *dart pub tool*? 
For example:
"The pub tool has commands for managing packages and for deploying packages and command-line apps. For general information about using the pub package manager, see How to use packages."

cheers,
Kevin

**Comment by kwalrath** [comment]
> I would like to start contributing to the [pub tool](https://dart.dev/tools/pub/cmd) sites.

Great! I'll assign `pub outdated` to you.

> I took a look at the website [pub outdated](https://dart.dev/tools/pub/cmd/pub-outdated) and have some open questions.
> 
> 1.) What should I do if a command output suggests using another command with the deprecated syntax?
> ...
> Should I open an issue to fix the static output information to change `pub outdated` to `dart pub outdated` ?

Yes, exactly. Please file an issue (or make sure an issue has already been filed) at https://github.com/dart-lang/sdk/issues.

> 2.) Is the whole tool rename or should I just focus on the correct syntax?
> In regarding to this question, if the sentence describes the _pub tool_, should it be changed to _dart pub tool_?
> For example:
> "The pub tool has commands for managing packages and for deploying packages and command-line apps. For general information about using the pub package manager, see How to use packages."

Let's still call it the _pub tool_. Only the syntax has changed.

**Comment by KevinRohn** [comment]
Hi @kwalrath,

I think you assigned an wrong user to the task. :)

**Comment by kwalrath** [comment]
> I think you assigned an wrong user to the task. :)

Oops! Fixed.

**Comment by KevinRohn** [comment]
Hi @kwalrath

if the first review is good enough let me know then I could also work on the other 'pub tool' commands.

**Comment by kwalrath** [comment]
Wonderful, will do!

**Comment by KevinRohn** [comment]
Ok, I checked the tasks, and for now you can assign following sites to me:

`src/tools/pub/cmd/pub-upgrade.md`
`src/tools/pub/cmd/pub-cache.md` (it's not on the list yet) 
`src/tools/pub/cmd/pub-get.md`

The other `pub tool` commands requires more time to test all the commands. 
I would like to do them step by step, after the above list.

**Comment by KevinRohn** [comment]
While checking the list I also noticed that the following entries are missing in the task list, but they are available on the `pub tools` site and the old syntax is still used there.
`[src/tools/pub/cmd/pub-cache.md](https://github.com/dart-lang/site-www/blob/master/src/tools/pub/cmd/pub-cache.md)`
`[src/tools/pub/cmd/pub-lish.md.md](https://github.com/dart-lang/site-www/blob/master/src/tools/pub/cmd/pub-lish.md)`
`[src/tools/pub/cmd/pub-run.md](https://github.com/dart-lang/site-www/blob/master/src/tools/pub/cmd/pub-run.md)`

**Comment by shubhanshu02** [comment]
Hi, @kwalrath! Yesterday, I was going through the web tutorial and tested it with the new dart tool. Can you assign me this [page](https://dart.dev/tutorials/web/get-started)? 
File: `src/_tutorials/web/get-started.md`

**Comment by kwalrath** [comment]
> Hi, @kwalrath! Yesterday, I was going through the web tutorial and tested it with the new dart tool. Can you assign me this [page](https://dart.dev/tutorials/web/get-started)?
> File: `src/_tutorials/web/get-started.md`

Done! Thank you, @shubhanshu02!

**Comment by shubhanshu02** [comment]
Hi @kwalrath! Can you assign me some more pages?

**Comment by sachin-cpp** [comment]
Hi @kwalrath ! I am a new contributor here. Can I work on [src/_guides](https://github.com/dart-lang/site-www/tree/master/src/_guides) ?

**Comment by kwalrath** [comment]
> Hi @kwalrath ! I am a new contributor here. Can I work on [src/_guides](https://github.com/dart-lang/site-www/tree/master/src/_guides) ?

You picked the hardest one to work on (/guides/packages) 😄 . But I'll assign all 4 of the /guides pages to you — the other 3 should be much easier!

**Comment by kwalrath** [comment]
> Hi @kwalrath! Can you assign me some more pages?

How about src/_tutorials/libraries/shared-pkgs.md? If there are any others you'd like to do, let me know. (I'm back from vacation, so I'll be more responsive now. :)

**Comment by shubhanshu02** [comment]
@kwalrath Ya, I'm good with that. I've nearly completed the changes and will make a PR in a few minutes. Any other file I could help with?

**Comment by kwalrath** [comment]
I've assigned src/tools/dart-devtools.md to @woinbo for an introductory PR.

**Comment by kwalrath** [comment]
@shubhanshu02 how about src/_tutorials/server/httpserver.md next?

**Comment by woinbo** [comment]
> I've assigned src/tools/dart-devtools.md to @woinbo for an introductory PR.

@kwalrath Thank you for this, let me start working on it.

**Comment by woinbo** [comment]
> I've assigned src/tools/dart-devtools.md to @woinbo for an introductory PR.

Hello @kwalrath, actually being my 1st open-source contribution I am not getting exactly what I have to do in this file src/tools/dart-devtools.md

**Comment by woinbo** [comment]
@kwalrath I have created a pull request. Kindly check it, please 😊 #2841

**Comment by Abhishek01039** [comment]
Hello @kwalrath 
Can you please assign `src/tools/pub/cmd/pub-global.md` to me?
Thanks

**Comment by chethanprabhakar** [comment]
Hello @kwalrath
Please assign `src/tools/pub/cmd/pub-downgrade.md`

**Comment by Sameerkash** [comment]
@kwalrath  I would like to be assigned to the last three

**Comment by kwalrath** [comment]
> Hey @kwalrath!! I would like to be assigned to the last three please.

Do you mean  src/tools/sdk/_linux.md, _windows.md, and _mac.md? (You have access to all three?)

**Comment by woinbo** [comment]
anything else for me @kwalrath

**Comment by woinbo** [comment]
may I work on src/tools/sdk/_windows.md i have access to windows @kwalrath

**Comment by Sameerkash** [comment]
Yes @kwalrath , okay you can assign me mac for now since I have primary access to it. Although I had windows +WSL2 and would probably have it back in some time. 

But you can go ahead and assign it to @woinbo

**Comment by kwalrath** [comment]
@Sameerkash you have _mac.md, and @woinbo you have _windows.md. Thank you!

**Comment by woinbo** [comment]
Hello @kwalrath, I did update _windows.md. Kindly check it, please. PR #2853

**Comment by woinbo** [comment]
Hello @kwalrath, I did PR #2854 for src/tools/pub/troubleshoot.md and updated this file to use new dart tool, kindly check it. Thank you!

**Comment by Sameerkash** [comment]
@kwalrath , I have some issues setting up the environment, followed all the steps for installing `nvm` and `rvm` as mentioned, but facing this error `travis_fold: command not found` when running `./tool/install.sh` and `./tool/before-install.sh  `

any pointers on what I might be doing wrong?

**Comment by Abhishek01039** [comment]
Hello @kwalrath 
seems like  `src/tools/dart-tool.md`  is done with the new dart tool.

**Comment by kwalrath** [comment]
> Hey @kwalrath , I have some issues setting up the environment, followed all the steps for installing `nvm` and `rvm` as mentioned, but facing this error `travis_fold: command not found` when running `./tool/install.sh` and `./tool/before-install.sh `
> 
> any pointers on what I might be doing wrong?

@Sameerkash your problems sound familiar, although I don't think I've seen them myself. Are you using bash? I think it's easier to use with our build than zsh is, although I don't know whether that's what causing this particular issue.

**Comment by Abhishek01039** [comment]
`src/tools/webdev.md` is done with this PR https://github.com/dart-lang/site-www/pull/2870.

**Comment by Sameerkash** [comment]
> > Hey @kwalrath , I have some issues setting up the environment, followed all the steps for installing `nvm` and `rvm` as mentioned, but facing this error `travis_fold: command not found` when running `./tool/install.sh` and `./tool/before-install.sh `
> > any pointers on what I might be doing wrong?
> 
> @Sameerkash your problems sound familiar, although I don't think I've seen them myself. Are you using bash? I think it's easier to use with our build than zsh is, although I don't know whether that's what causing this particular issue.

I'm using bash now, everything executed successfully, but still, some dart dependencies are not resolving although there's no error while doing pub get

**Comment by woinbo** [comment]
Hey @kwalrath, I did update src/tools/pub/cmd/pub-uploader.md PR no #2883

**Comment by shubhanshu02** [comment]
@kwalrath I can help with the `src/tools/sdk/_linux.md`, if it is not assigned to anyone yet. I use Linux as my primary operating system.

**Comment by kwalrath** [comment]
@shubhanshu02 _linux.md is yours. Thank you!

**Comment by kwalrath** [comment]
I've updated the assignment list, and I removed some assignments from people who don't seem to be active any more.

If you'd like to talk about any of this, I'm [on Flutter's discord server](https://github.com/flutter/flutter/wiki/Chat) in the #hackers-devrel channel. (I've updated the CONTRIBUTING.md file to have the same link.)

**Comment by Sameerkash** [comment]
@kwalrath I'm not getting the packages resolved. 

I don't need to do that since its just doc update right ? Can I go ahead and push it without actually building the site ?

**Comment by kwalrath** [comment]
> @kwalrath I'm not getting the packages resolved.
> 
> I don't need to do that since its just doc update right ? Can I go ahead and push it without actually building the site ?

Yes, just go ahead and push. I'll stage your changes to make sure they work.

**Comment by Sameerkash** [comment]
@kwalrath I didn't go through the file before focusing on all of this 😅. 
And now that I have, I think there are no changes to be made in `src/tools/sdk/_mac.md`, it has instructions to install dart-sdk using homebrew and no references to the dart cli tools. Hence I believe it can be safely closed. 

And I've gone through `src/_includes/get-sdk.md` and has the same instructions to install `dart-sdk` and no references to the cli tools, although #2883 can be added for Windows instructions. 

I've spotted references to `dartfmt` and  `pub` in `src/_guides/language/formatting.md` and  ` src/_guides/packages.md`, so if I can be assigned to that, I can go ahead and make the new dart tool references.

If you do think there are any changes to be made in the ones I mentioned above. Please do let me know. I'll make it asap.

**Comment by kwalrath** [comment]
> @kwalrath I didn't go through the file before focusing on all of this 😅.
> And now that I have, I think there are no changes to be made in `src/tools/sdk/_mac.md`, it has instructions to install dart-sdk using homebrew and no references to the dart cli tools. Hence I believe it can be safely closed.

Sounds good. @shubhanshu02 found the same for _linux.md, and it makes sense that the installation instructions are unlikely to contain references to Dart SDK commands. (When compiling that list of files, I just looked for command-line formatting, not for specific commands.)

> And I've gone through `src/_includes/get-sdk.md` and has the same instructions to install `dart-sdk` and no references to the cli tools, although #2883 can be added for Windows instructions.

Thanks for checking that. #2883 doesn't look related, though... Could you check the PR/issue number?

> I've spotted references to `dartfmt` and `pub` in `src/_guides/language/formatting.md` and ` src/_guides/packages.md`, so if I can be assigned to that, I can go ahead and make the new dart tool references.

Consider them yours. Thanks!

I guess we should do a final grep pass for all occurrences of the old command names, whether or not they're in terminals.

**Comment by Sameerkash** [comment]
Hey @kwalrath, 
- [x] `src/_includes/get-sdk.md` at  #2906
- [x]  `src/_guides/language/formatting.md` at  #2904
- [x] `src/_guides/packages.md` at #2905

have been updated, could you please review them?

**Comment by shubhanshu02** [comment]
@kwalrath Can you assign me the ` src/_guides/libraries/c-interop.md` or `src/tools/pub/cmd/pub-deps.md`?

**Comment by kwalrath** [comment]
> @kwalrath Can you assign me the ` src/_guides/libraries/c-interop.md`

You've got it. Thanks!

**Comment by woinbo** [comment]
Hello @kwalrath, sorry I was out for some reasons!! Any thing for me??

**Comment by kwalrath** [comment]
@woinbo how about src/tools/experiment-flags.md? That's one that could probably use some real testing, since the examples are either old or maybe not real. :)

**Comment by woinbo** [comment]
@kwalrath please review these PR #2865 #2866 #2867, these are in the pull request tab from last 25 days. You haven't checked them? I will see src/tools/experiment-flags.md today or tomorrow. Thank you!!

**Comment by kwalrath** [comment]
Sorry for the backlog, @woinbo... Things are crazy busy here right now. I do intend to take a look. Thanks for taking a look at experiment-flags!

**Comment by bharat-biradar** [comment]
@kwalrath I can take them

**Comment by bharat-biradar** [comment]
@parlough is there anything that needs to be changed in [experiment-flags.md](https://github.com/dart-lang/site-www/blob/master/src/tools/experiment-flags.md). I couldn't find anything

**Comment by woinbo** [comment]
> @parlough is there anything that needs to be changed in [experiment-flags.md](https://github.com/dart-lang/site-www/blob/master/src/tools/experiment-flags.md). I couldn't find anything

I too looked at this file but found nothing which should be corrected, Everything is perfect and fine. Thank you!

**Comment by kwalrath** [comment]
At a glance, `$ dart --enable-experiment=super-mixins,no-slow-checks bin/main.dart` needs to be changed to use `dart run`. That might be all.

**Comment by bharat-biradar** [comment]
@woinbo As mentioned in the [comment](https://github.com/dart-lang/site-www/issues/2617#issuecomment-775352675) should a new example be added? Or is it fine to just port the existing docs for now?

**Comment by bharat-biradar** [comment]
@kwalrath Except for this [installation script](https://github.com/dart-lang/site-www#3-run-installation-scripts) anything else needs to be updated?

**Comment by kwalrath** [comment]
> @kwalrath Except for this [installation script](https://github.com/dart-lang/site-www#3-run-installation-scripts) anything else needs to be updated?

https://dart.dev/tools/pub/cmd/pub-deps needs to be updated to match the other pub subcommands pages (e.g. https://dart.dev/tools/pub/cmd/pub-cache).

**Comment by bharat-biradar** [comment]
@kwalrath I have opened a [PR](https://github.com/dart-lang/site-www/pull/3118) for pub deps.
I found some installation scripts that use pub commands please see if they also need to be updated
- [site-shared/tools/check-links.sh](https://github.com/dart-lang/site-shared/blob/824f80fa9c91f49299c02ac0a43920a65877bba4/tool/check-links.sh#L89)
- [sire-shared/tools/before-install.sh](https://github.com/dart-lang/site-shared/blob/824f80fa9c91f49299c02ac0a43920a65877bba4/tool/before-install.sh#L12)

**Comment by kwalrath** [comment]
> @kwalrath I have opened a [PR](https://github.com/dart-lang/site-www/pull/3116) for pub deps.

Thank you! I've merged it.

> I found some installation scripts that use pub commands please see if they also need to be updated
> 
> * [site-shared/tools/check-links.sh](https://github.com/dart-lang/site-shared/blob/824f80fa9c91f49299c02ac0a43920a65877bba4/tool/check-links.sh#L89)
> * [sire-shared/tools/before-install.sh](https://github.com/dart-lang/site-shared/blob/824f80fa9c91f49299c02ac0a43920a65877bba4/tool/before-install.sh#L12)

Yes, those should be updated, too. Want to do it?

**Comment by bharat-biradar** [comment]
@kwalrath Yes, I'll update them.

**Comment by kwalrath** [comment]
@bharat-biradar no need to update site-shared... looks like @mit-mit has just done it in https://github.com/dart-lang/site-shared/pull/129.

**Comment by kwalrath** [comment]
I'm going to update site-www to use the new site-shared now, and then... I think we'll be done!
